src/dbt_osmosis/core/osmosis.py

```diff
@@ -20,6 +20,7 @@
 )
 
 import ruamel.yaml
+from dbt.contracts.results import ColumnMetadata
```

Comment:
When was this introduced? Currently we are [probably] compatible from dbt 1.0 -> 1.5 at least, probably 1.6 too. We should be considerate of this and if the api is not available -- do a try/except and put a mock dataclass up if it doesnt exist. Otherwise bump min dbt version, either way.

Reply:
It looks like it's there
https://github.com/dbt-labs/dbt-core/blob/ec1f609f3ee8fb3e6e7393606085e121c10876cc/core/dbt/contracts/results.py#L420